models/Model.js

```diff
@@ -105,6 +105,21 @@ updateArticleSelect = (responseBody, article_id) => {
     });
 };
 
+deleteCommentModel = (comment_id) => {
+  if (isNaN(Number(comment_id))) {
+    return Promise.reject({ status: 400, msg: "Bad Request - Invalid ID" });
+  }
```

Comment:
You don't need to check if the comment_id is a number, psql will reject it and give you an error itself!